tests/functional/spec/interstitial.spec.js

```diff
@@ -0,0 +1,29 @@
+import chai from 'chai'
+import {startQuestionnaire} from '../helpers'
+import InterstitialMultipleChoiceWithOtherPage from '../pages/surveys/interstitial/multiple-choice.page'
+import InterstitialPage from '../pages/surveys/interstitial/interstitial.page'
+import SummaryPage from '../pages/surveys/interstitial/summary.page'
+
+const expect = chai.expect
+
+describe('Interstitial Pages', function () {
+
+    var interstitial_schema = 'test_interstitial_page.json';
+
+
+    it('Given I am completing a survey with an interstitial page, when I arrive at the interstitial page, then i should be able to continue the survey.', function () {
+        //Given
+        startQuestionnaire(interstitial_schema)
+        InterstitialMultipleChoiceWithOtherPage.clickOther().setOtherInputField('Cereal').submit()
+
+        // When
+        expect(InterstitialPage.getTitle()).to.contain('Breakfast Interstitial Page')
+
+        //Then
+        InterstitialPage.submit()
```

Comment:
Are we doing more than we need to in this test by going to the summary page and checking it? Are we covered by other tests on that?